def/conference.go

```diff
@@ -1,7 +1,20 @@
 package def
 
-// Conference is an event
-// gorm: "true"
+import "time"
+
+// Conference is a brand like GopherCon
 type Conference struct {
-	Name string
+	ID     uint
+	Name   string
+	Events []Event
+}
+
+// Event is an instance like GopherCon 2020
+type Event struct {
+	ID        uint
+	Name      string
+	Slug      string
+	StartDate time.Time
+	EndDate   time.Time
+	Location  string
```

Comment:
Perhaps we want this to be a type? I imagine this always being a Venue, and a geographical ref like city/state.

Reply:
thought this through and came to the conclusion that these structs should just represent basic metadata. Public facing descriptions of this metadata should be editable by non-technical users.  Added #pace14 to capture this.